packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -253,15 +254,44 @@ export const validateCart = async (
   const {
     clients: { commerce },
     loaders: { skuLoader },
+    headers
   } = ctx
 
   // Step1: Get OrderForm from VTEX Commerce
   const orderForm = await getOrderForm(orderNumber, session, ctx)
 
+  
+  // Validate if Session's cookie exists
+  const cookieSession = getCookie('vtex_session', headers.cookie) 
+
+  if (cookieSession && enableOrderFormSync === true) {
+
+    const {namespaces} =  await commerce.getsessionorder()  
+    const orderFormIdSession = namespaces.checkout?.orderFormId?.value 
+
+    // In the case of divergence between session cookie and indexdb update the order form
+    if (orderNumber != orderFormIdSession && orderFormIdSession!= undefined){
```

Comment:
hmm, what about extract it to a `const` that represents what the comment say, maybe:
```suggestion
    const isSessionOrderFormSync = orderFormIdSession && (orderFormIdSession == orderNumber)
    if (!isSessionOrderFormSync){
```